go-cosmwasm/api/lib_test.go

```diff
@@ -114,7 +114,7 @@ func TestInstantiate(t *testing.T) {
 	require.NoError(t, err)
 	msg := []byte(`{"verifier": "fred", "beneficiary": "bob"}`)
 
-	res, cost, err := Instantiate(cache, id, params, msg, &gasMeter, &store, api, &querier, 100000000)
+	res, cost, err := Instantiate(cache, id, params, msg, &gasMeter, &store, api, &querier, 100000000, nil, nil)
```

Comment:
does just adding `nil` here work correctly? seems like it would fail in HW mode

Reply:
It was only to reduce compilation errors for now. Will fix later of course